README.md

```diff
@@ -84,6 +85,38 @@ them against local or cloud-based resources seamlessly.
 
 ![Enhanced Python Support](docs/source/images/python-runner.png)
 
+#### Reusable Code Snippets (Experimental)
+
+Elyra supports a beta version of the **Code Snippet** feature. 
+This allows users to add custom pieces of code that can be reused, making programming in JupyterLab more efficient
+by reducing repetitive work.
+
+![Code Snippets](docs/source/images/code-snippets.png)
+
+Currently, code snippets can be added as json files in your local Jupyter Data directory under `metadata/code-snippets` subdirectory.
+Here is a content sample of a snippet json file:
+```json
+{
+	"schema_name": "code-snippet",
+	"display_name": "is_even",
+	"metadata": {
+		"description": "Check if number is even",
+		"language": "python",
+		"code": [
+			"def is_even(num):",
+			"   return num % 2 == 0"
+		]
+	}
+}
+```
+NOTE: code field content is split by line
+
+The extension then retrieves the data and displays a list of code snippets,
+which can be individually expanded on click for easy access of the code content.
+The snippet can be manually copied from the viewer into the editor.
+
+![Code Snippet Sample](docs/source/images/code-snippet-expanded.png)
+
```

Comment:
I still think this is too much information in the README.  I would suggest only showing the code-snippet-expanded image, followed by a statement like:  "For more information see Elyra Code Snippets" where "Elyra Code Snippets" is a link to code-snippets.html.